docs/.vuepress/sidebar.js

```diff
@@ -114,7 +114,8 @@ module.exports = {
       collapsable: false,
       children: [
         ['cookbook/building-navigation.md', 'Building site navigation'],
-        ['cookbook/html-conversion.md', 'Converting an HTML template']
+        ['cookbook/html-conversion.md', 'Converting an HTML template'],
+        ['cookbook/creating-custom-schema-fields.md', 'Creating custom schema fields']
```

Comment:
How will we distinguish between this article, and the existing article:

https://v3.docs.apostrophecms.org/guide/custom-schema-field-types.html

I like your two-part structure, but I think my existing article does a good job of covering the other common use case, maybe more common use case, where the reason for a custom field is a desire for truly custom UI. We should distinguish the two major use cases clearly (creating a custom schema field with a unique user interface, versus creating a custom schema field which reuses other field types, possibly in addition to unique elements).

Reply:
Changed.